guides/common/modules/proc_deploying_an_application_instance.adoc

```diff
@@ -0,0 +1,50 @@
+[id="{context}_deploying_an_application_instance"]
+= Deploying an Application Instance
+
```

Comment:
Introduce what the procedure does and why to do it, and any prerequisites.

Reply:
Would it be useful to list configured "Ansible playbook", "application definition", and "application instance" as a requirement?

So far, I've added minimal "one sentence" introductions to each procedure.

Reply:
There is a documentation book called "Every Page is Page One". I am unsure if I read it, because I think that's the entire point of the book. So, if you were to treat the fact that people won't read the docs like a book, but might google and end up on this page, on this section, it would be helpful to have a list of prereqs with links to the relevant sections.

Reply:
I've added a list of requirements to the introduction section.